src/pybind/mgr/dashboard/requirements-lint.txt

```diff
@@ -9,3 +9,5 @@ flake8-colors==0.1.6; python_version >= '3'
 rstcheck==3.3.1; python_version >= '3'
 autopep8; python_version >= '3'
 pyfakefs; python_version >= '3'
+isort==5.5.3
```

Comment:
This change breaks `make check` in octopus:

```
ERROR: Cannot install -r requirements-lint.txt (line 1) and isort==5.5.3 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested isort==5.5.3
    pylint 2.3.1 depends on isort<5 and >=4.2.5

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

Reply:
Thanks for your help, @smithfarm!
@bk201 do you think we should only fix `ceph_service.py` instead of cherry-picking the whole commit or should we try it with a different version of isort?

Reply:
I also tend to think the linter bits of this changeset could be dropped, since they would presumably not be needed to fix the issue at hand.

Reply:
> Thanks for your help, @smithfarm!
> @bk201 do you think we should only fix `ceph_service.py` ([#37972 (comment)](https://github.com/ceph/ceph/pull/37972#discussion_r524046941)) instead of cherry-picking the whole commit or should we try it with a different version of isort?

I think it's OK to ignore the isort.